less/variables.less

```diff
@@ -483,7 +483,8 @@
 @jumbotron-color:                inherit;
 @jumbotron-bg:                   @gray-lighter;
 @jumbotron-heading-color:        inherit;
-@jumbotron-font-size:            ceil((@font-size-base * 1.5));
+@jumbotron-heading-font-size:    ceil((@font-size-base * 1.5));
```

Comment:
phooey - yeah you're correct. Should I just scrap this commit/request and submit a new one, or make another commit and add it to the request?